docs/components.md

```diff
@@ -21,28 +21,27 @@ KubeVirt consists of a set of services:
 
 ## Example flow: Create and Delete a VM
 
-<!-- FIXME: wrong in a lot of ways and therefore misleading -->
-
-    User       Virt API      VM TPR    Virt Controller    k8s      VM Pod
-    createVM  --> |                           |            |
-                  o create --> +              |            |
-                  |            | <~ create ~> ?            |
-                  |            |              |            |
-                  |            |              o create --> o ------> +
-                  |            |              |            |         |
-                  |            | <---------------------- get VM Spec o
-                  |            o ----------------------------------> |
-                  |            |              |            |         o defineVM()
-                  |            |              |            |         o watchVM()
-                  |            |              |            |         |
-    deleteVM -->  o delete --> *              |            |         |
-                  |              <~ delete ~> ?            |
-                  |                           |            |         |
-                  |                           o delete --> o ------> *
-                  |                           |            |
-                  :                           :            :
-
-    Legend: ?: Event notification
+    Client        Virt API      VM TPR  Virt Controller    k8s      VM Pod
+    ------------- ------------- ------- ------------------ -------- ------
+
+                  listen <------------- WATCH /vms
+                                           |
+    POST /vms ----> validate               |
+                    create ---> VM ---> observe
+                      |         |          v
+                      |         |       POST /pods ----> validate
+                      |         |          |             create --> Pod
+                      |         |          |                         |
+                      |         |          |                         v
+                      | <-----------------------------------------> GET /vms
+                      |         |          |                        defineVM
+                      |         |          |                        watchVM
+                      |         |          |                         |
+    DELETE /vms --> validate    |          |                         |
```

Comment:
Here it would also be that we passthrough the kall to k8s which then deletes the tpr.

So this needs to be fixed, but …